.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,32 @@
+name: CI (windows)
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-ci:
+    name: Run
+
+    runs-on: windows-2019
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup NuGet
+        uses: NuGet/setup-nuget@v1
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1
+
+      - name: Install dependencies
+        run: |
+          nuget restore Twingly.Search.sln
+          nuget install NUnit.Console -Version 3.9.0 -OutputDirectory packages
```

Comment:
Since this is a new file, it makes sense to me to use latest available: https://www.nuget.org/packages/NUnit.Console/3.11.1

Reply:
(I know it's 3.9.0 in Mono, but then it was just a mindless port from Travis 😄)

Reply:
All right, I'll update